share/locale/CMakeLists.txt

```diff
@@ -18,166 +18,37 @@
-      mscore_it.qm
-      mscore_ja.qm
-      mscore_ka.qm
-      mscore_ko.qm
-      mscore_lt.qm
-      mscore_lv.qm
-      mscore_ml.qm
-      mscore_mn_MN.qm
-      mscore_nl.qm
-      mscore_nb.qm
-      mscore_nn.qm
-      mscore_pl.qm
-      mscore_pt.qm
-      mscore_pt_BR.qm
-      mscore_ro.qm
-      mscore_ru.qm
-      mscore_sk.qm
-      mscore_sl.qm
-      mscore_sr.qm
-      mscore_sr_RS.qm
-      mscore_sv.qm
-      mscore_th.qm
-      mscore_tr.qm
-      mscore_uk.qm
-      mscore_uz@Latn.qm
-      mscore_vi.qm
-      mscore_zh_CN.qm
-      mscore_zh_TW.qm
-      languages.xml
-      qt_ar.qm
-      qt_ca.qm
-      qt_cs.qm
-      qt_da.qm
-      qt_de.qm
-      qt_el.qm
-      qt_es.qm
-      qt_eu.qm
-      qt_fa.qm
-      qt_fi.qm
-      qt_fr.qm
-      qt_gl.qm
-      qt_he.qm
-      qt_hu.qm
-      qt_id.qm
-      qt_it.qm
-      qt_ja.qm
-      qt_ko.qm
-      qt_lt.qm
-      qt_nb.qm
-      qt_nl.qm
-      qt_nl_BE.qm
-      qt_pl.qm
-      qt_pt.qm
-      qt_pt_BR.qm
-      qt_ro.qm
-      qt_ru.qm
-      qt_sk.qm
-      qt_sv.qm
-      qt_tr.qm
-      qt_uk.qm
-      qt_vi.qm
-      qt_zh_CN.qm
-      qt_zh_TW.qm
-      instruments_af.qm
-      instruments_ar.qm
-      instruments_ar_DZ.qm
-      instruments_ar_EG.qm
-      instruments_ar_SD.qm
-      instruments_ast.qm
-      instruments_be.qm
-      instruments_bg.qm
-      instruments_ca.qm
-      instruments_ca@valencia.qm
-      instruments_cs.qm
-      instruments_cy.qm
-      instruments_da.qm
-      instruments_de.qm
-      instruments_el.qm
-      instruments_en_GB.qm
-      instruments_en_US.qm
-      instruments_eo.qm
-      instruments_es.qm
-      instruments_et.qm
-      instruments_eu.qm
-      instruments_fa.qm
-      instruments_fi.qm
-      instruments_fo.qm
-      instruments_fr.qm
-      instruments_ga.qm
-      instruments_gd.qm
-      instruments_gl.qm
-      instruments_he.qm
-      instruments_hi_IN.qm
-      instruments_hr.qm
-      instruments_hu.qm
-      instruments_hy.qm
-      instruments_id.qm
-      instruments_it.qm
-      instruments_ja.qm
-      instruments_ka.qm
-      instruments_ko.qm
-      instruments_lt.qm
-      instruments_lv.qm
-      instruments_ml.qm
-      instruments_mn_MN.qm
-      instruments_nl.qm
-      instruments_nb.qm
-      instruments_nn.qm
-      instruments_pl.qm
-      instruments_pt.qm
-      instruments_pt_BR.qm
-      instruments_ro.qm
-      instruments_ru.qm
-      instruments_sk.qm
-      instruments_sl.qm
-      instruments_sr.qm
-      instruments_sr_RS.qm
-      instruments_sv.qm
-      instruments_th.qm
-      instruments_tr.qm
-      instruments_uk.qm
-      instruments_uz@Latn.qm
-      instruments_vi.qm
-      instruments_zh_CN.qm
-      instruments_zh_TW.qm
-      DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}locale
-      )
+install(DIRECTORY
+    ./
+    DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}locale
+    FILES_MATCHING
+    REGEX ".*\\.qm"
+    REGEX "qt_blank\\.qm" EXCLUDE
```

Comment:
This could probably removed too, just need to find out why it does generated in the first place, regardless not being mentioned in build/gen-qt-projectfile or build/gen-qt-projectfile.bat.